<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Tvl-2" name="mainPage" tests="28" failures="3" timestamp="3 Apr 2014 07:55:59 GMT" time="551.756" errors="0">
  <testcase name="_01_T_FacebookAuthorisation" time="16.232" classname="Tests._01_FacebookAuthorisation"/>
  <testcase name="_02_T_nputFieldText" time="1.353" classname="Tests._02_InputFieldText"/>
  <testcase name="_03_T_PopUpOnLogin" time="1.09" classname="Tests._03_PopUpOnLogin"/>
  <testcase name="_04_T_PopUpOnDetectMy" time="2.251" classname="Tests._04_PopUpOnDetectMy"/>
  <testcase name="_05_T_PopUpOnDetectMy_02" time="17.66" classname="Tests._05_PopUpOnDetectMy_02"/>
  <testcase name="_06_T_NameRequired" time="3.269" classname="Tests._06_NameRequired"/>
  <testcase name="_07_T_IllegalMail" time="3.415" classname="Tests._07_IllegalMail"/>
  <testcase name="_08_T_RoleInput" time="1.231" classname="Tests._08_RoleInput"/>
  <testcase name="_09_T_DetectLink" time="1.005" classname="Tests._09_DetectLink"/>
  <testcase name="_10_T_ChemistryScoreButtonPopUp" time="11.466" classname="Tests._10_ChemistryScoreButtonPopUp"/>
  <testcase name="_11_T_DeleteEl" time="18.472" classname="Tests._11_DeleteEl"/>
  <testcase name="_12_T_SaveTeamData" time="21.639" classname="Tests._12_SaveTeamData"/>
  <testcase name="_13_T_LockEdit" time="10.776" classname="Tests._13_LockEdit"/>
  <testcase name="_14_T_QqForMyself" time="2.425" classname="Tests._14_QqForMyself"/>
  <testcase name="_15_T_QqDiscByMyself" time="4.292" classname="Tests._15_QqDiscByMyself"/>
  <testcase name="_16_T_FwBwArrows" time="2.812" classname="Tests._16_FwBwArrows"/>
  <testcase name="_17_T_WarningPopUpTestPage" time="5.764" classname="Tests._17_WarningPopUpTestPage"/>
  <testcase name="_18_T_SocTypeAssert" time="5.138" classname="Tests._18_SocTypeAssert"/>
  <testcase name="_09_T_CommonElementsGraph" time="13.029" classname="Tests._19_CommonElementsGraph"/>
  <testcase name="_20_T_SelectVertex" time="12.301" classname="Tests._20_SelectVertex">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.assertNotEquals(Assert.java:854)
	at org.testng.Assert.assertNotEquals(Assert.java:859)
	at Tests._20_SelectVertex._20_T_SelectVertex(_20_SelectVertex.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- _20_T_SelectVertex -->
  <testcase name="_21_T_ClickOnLine" time="17.064" classname="Tests._21_ClickOnLine">
    <failure type="java.lang.AssertionError" message="expected [ity] but found [Quasi-identity]">
      <![CDATA[java.lang.AssertionError: expected [ity] but found [Quasi-identity]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at Tests._21_ClickOnLine._21_T_ClickOnLine(_21_ClickOnLine.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- _21_T_ClickOnLine -->
  <testcase name="_22_T_PopUpOnVertex" time="11.469" classname="Tests._22_PopUpOnVertex"/>
  <testcase name="_23_T_DeletePersonFromGraph" time="19.387" classname="Tests._23_DeletePersonFromGraph"/>
  <testcase name="_24_T_AllDeletePersonFromGraph" time="30.21" classname="Tests._24_AllDeletePersonFromGraph">
    <failure type="org.openqa.selenium.TimeoutException" message="Timed out after 10 seconds waiting for visibility of [[ChromeDriver: chrome on XP (d7513e3a8d72bda8e9b1c00359e708c4)] -&amp;gt; xpath: //*[@id=&amp;apos;dc&amp;apos;]/div/div[2]/div/span]
Build info: version: &amp;apos;2.41.0&amp;apos;, revision: &amp;apos;3192d8a&amp;apos;, time: &amp;apos;2014-03-27 17:18:15&amp;apos;
System info: host: &amp;apos;Tvl-2&amp;apos;, ip: &amp;apos;169.254.199.30&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.8.0&amp;apos;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 10 seconds waiting for visibility of [[ChromeDriver: chrome on XP (d7513e3a8d72bda8e9b1c00359e708c4)] -> xpath: //*[@id='dc']/div/div[2]/div/span]
Build info: version: '2.41.0', revision: '3192d8a', time: '2014-03-27 17:18:15'
System info: host: 'Tvl-2', ip: '169.254.199.30', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:259)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at pages.MainPage.clickAndTestAll(MainPage.java:271)
	at Tests._24_AllDeletePersonFromGraph._24_T_AllDeletePersonFromGraph(_24_AllDeletePersonFromGraph.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- _24_T_AllDeletePersonFromGraph -->
  <testcase name="_25_T_ClickOnVertex" time="20.026" classname="Tests._25_ClickOnVertex"/>
  <testcase name="_26_T_" time="0.064" classname="Tests._25_ClickOnVertex"/>
  <testcase name="_26_T_AddPersonFromGraph" time="14.25" classname="Tests._26_AddPersonFromGraph"/>
  <testcase name="_27_T_LineSwitch" time="56.256" classname="Tests._27_LineSwitch"/>
</testsuite> <!-- mainPage -->
